PAWSModules/Package.swift

```diff
@@ -73,6 +77,27 @@ let package = Package(
         .target(
             name: "PAWSSharedContext",
             dependencies: []
+        ),
+        .target(
+            name: "PAWSLandingScreen",
+            dependencies: [],
+            resources: [
+                .process("Resources")
+            ]
+        ),
+        .target(
+            name: "PAWSNotificationScreen",
+            dependencies: [],
+            resources: [
+                .process("Resources")
+            ]
+        ),
+        .target(
+            name: "PAWSRegisterScreen",
+            dependencies: [],
```

Comment:
You can remove empty dependency definitions if you don't use a dependency:
```suggestion
            name: "PAWSRegisterScreen"
```